cpp/tensorrt_llm/kernels/trtllmGenKernels/fmha/fmhaRunnerParams.h

```diff
@@ -240,6 +242,8 @@ struct TllmGenFmhaRunnerParams
     void* oPtr;
     // The output scaling factor buffer.
     void* oSfPtr;
+    // The spec-decoding generation lengths.
+    int const* spec_decoding_generation_lengths;
```

Comment:
I would prefer `seqlensQPtr` to make it more general. please follow the camel naming style here. same for other parameters. Thanks.